packages/@ama-sdk/generator-sdk/project.json

```diff
@@ -38,7 +38,9 @@
         "outputPath": "packages/@ama-sdk/generator-sdk/dist",
         "assets": [
           "packages/@ama-sdk/generator-sdk/src/**/swagger-codegen-cli.jar",
+          "packages/@ama-sdk/generator-sdk/src/**/config/*.json",
           "packages/@ama-sdk/generator-sdk/src/generators/**/target/*.jar",
+          "packages/@ama-sdk/generator-sdk/src/generators/**/target/**/*",
```

Comment:
what are the other files (in addition of the jar) do you need?

Reply:
none in fact